test/gst-msdk/util.py

```diff
@@ -32,6 +32,7 @@ def get_supported_format_map():
     "P010"  : ("p010-10le", "P010_10LE"),
     "P210"  : ("i422-10le", "I422_10LE"),
     "P410"  : ("y444-10le", "Y444_10LE"),
+    "Y410"  : ("y410", "Y410"), #y410: entry for GstVideoFormat properties such as vaapipostproc ; Y410: is entry for capsfilter
```

Comment:
Can you provide a "general" comment for the function instead, please?  The first entry is for element properties, whereas the second entry is for caps filters.  This applies to all of these formats in the map.

Reply:
updated